source/codegen/metadata/nidmm/config.py

```diff
@@ -16,7 +16,7 @@
     },
     'custom_types': [
     ],
-    'driver_name': 'NI-DMM',
+    'driver_name': 'NI-Dmm',
```

Comment:
This one should stay "NI-DMM". The "Dmm" casing only applies to C++/C# class names.

Reply:
changed it